generate.js

```diff
@@ -43,11 +43,12 @@ const template = `
   </head>
   <body>
     <h1>GCI Current Leaders</h1>
-    <i>Names are listed alphabetically</i>
+    <i>Participants names are listed alphabetically</i>
```

Comment:
Something to consider...

These values are not normal 'names', as many are usernames.

> The participants are listed alphabetically using on their "display name".

?

It is a bit too wordy...

Reply:
Maybe just insert "display" after "participants" ?